static/configs/PHPStrom_MJ_CodeStyle.xml

```diff
@@ -0,0 +1,192 @@
+<code_scheme name="Project" version="173">
```

Comment:
@Alex-DA rename the file to be more generic and not specific to PhpStorm. I would called it *Jetbrains_MJ_CodeStyle.xml*

Reply:
Fixed in https://github.com/mobilejazz/harmony-reference/pull/82/commits/17ef30f91a245bf29b8a8b37ac9c8d4f421e00c0